nomos-libp2p/src/behaviour/mod.rs

```diff
@@ -7,7 +7,6 @@ use std::error::Error;
 
 use cryptarchia_sync::ChainSyncError;
 use libp2p::{identity, kad, swarm::NetworkBehaviour, PeerId};
-use thiserror::Error;
```

Comment:
why remove this?

Reply:
Because that is what I always do. Fewer dependencies reduce compilation times as well as the surface area of supply chain attacks. Not only for `thiserror` but also for any other applicable crate.

But that is only me. If you want I can revert this change, that is not a problem.